node/node.go

```diff
@@ -1060,6 +1067,16 @@ func (n *Node) getLatestValidators() (map[uint32][]*state.ValidatorInfo, map[str
 	return validators, nil, nil
 }
 
+// EncodeAddressPubkey will encode the provided address public key bytes to string
+func (n *Node) EncodeAddressPubkey(pk []byte) string {
+	return n.addressPubkeyConverter.Encode(pk)
```

Comment:
I think a nil check for the addressPubkeyConverter is missing

Reply:
added, propagated all the way through the api route.

Reply:
ok